cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,36 @@
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+import HomeAndCataloguePageObject from
+  '../support/pages/homeСatalogue.pageObject';
+import CartPageObject from '../support/pages/cart.pageObject';
+import ContactFormPageObject from '../support/pages/contactForm.pageObject';
 
+const homeAndСatalogue = new HomeAndCataloguePageObject();
+const cartPage = new CartPageObject();
+const contactForm = new ContactFormPageObject();
+describe('Demoblaze app', () => {
+  before(() => {
+    homeAndСatalogue.visit();
   });
 
-  it('', () => {
+  it('checking the ability to add and order goods', () => {
+    homeAndСatalogue.clickOnCategory('Laptop');
+    homeAndСatalogue.clickOnProduct('Sony vaio i7');
+    homeAndСatalogue.clickOnTheButton('Add to cart');
+    homeAndСatalogue.succsessfulAllert();
+    homeAndСatalogue.clickOnLink('Cart');
+
+    cartPage.assertAboutProduct('Sony vaio i7');
+    cartPage.clickOnTheButton('Place Order');
 
+    contactForm.typeName('Ivan');
+    contactForm.typeCountry('Ukraine');
+    contactForm.typeCity('Lviv');
+    contactForm.typeCreditCart(4441123445678910);
+    contactForm.typeMonth('May');
+    contactForm.typeYear(2023);
+    contactForm.clickOnTheButton('Purchase');
+    contactForm.assertData('Ivan', 4441123445678910);
```

Comment:
The 'assertData' function probably should take more arguments to assert all the form data entered, not only name and credit card number.